primitives/src/proxy_type.rs

```diff
@@ -36,6 +37,11 @@ pub enum ProxyType {
     CancelProxy,
     Governance,
     Staking,
+    CreateMarket,
+    ReportOutcome,
```

Comment:
We should have a proxy for disputing.

Reply:
https://github.com/zeitgeistpm/zeitgeist/pull/1038/commits/b80ae75c2da1bc7ce687cb5e2a5aa6d1aef2d37c